katcp/server.py

```diff
@@ -34,9 +34,12 @@
 from .sampling import format_inform_v5, format_inform_v4
 from .core import (SEC_TO_MS_FAC, MS_TO_SEC_FAC, SEC_TS_KATCP_MAJOR,
                    VERSION_CONNECT_KATCP_MAJOR, DEFAULT_KATCP_MAJOR)
-from .version import VERSION, VERSION_STR
 from .kattypes import (request, return_reply)
 
+# 'import katcp' so that we can use katcp.__version__ later
+# we cannot do this: 'from . import __version__' because __version__
+# does not exist at this stage
+import katcp
```

Comment:
Would it not work to change the order of statements in __init__.py such that __version__ is generated before it imports from other submodules? Otherwise we have a bit of a circular import which can become fun IIRC. In any case, I don't have strong feelings about this if what you did works already, just a comment.